biosys/apps/main/models.py

```diff
@@ -193,6 +194,10 @@ def has_object_destroy_permission(self, request):
     def centroid(self):
         return self.geometry.centroid if self.geometry else None
 
+    @property
+    def extent(self):
+        return self.geometry.extent if self.geometry else None
```

Comment:
Question: how `None` is represented in a `serializers.ListField`? Empty list or None?

Reply:
Comes out None